score-client/src/main/bin/score-client

```diff
@@ -35,10 +35,16 @@ fi
 
 if [[ "$_java" ]]; then
     version=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
-    if [[ "$version" > "1.8" ]]; then
-        :
+
+    regex="^1?\.." #matches 1.xx java versioning
+    if [[ $version =~ $regex ]]; then 
+        versionMajor=$(echo $version | awk -F '.' '{print $2}') 
     else
-        echo Java 1.8 required for SCORe Client. Current version is $version
+	versionMajor=$(echo $version | awk -F '.' '{print $1}') 
+    fi
+    
+    if [[ $versionMajor -lt 11 ]]; then
+        echo Java 11 required for SCORe Client. Current version is $version
```

Comment:
you can merge of all this to say, if the regex is not matches, then error out and indicate the version that caused the error